src/dl_types.h

```diff
@@ -89,8 +89,8 @@ typedef enum
 
 static const uintptr_t DL_NULL_PTR_OFFSET[2] =
```

Comment:
I guess this can be removed now :)
But we should probably do that in a separate change as this is alreayd really big (as you said :) )

Reply:
I would like to make that incrementally, yes.

Additionally, my own master have already grown past the pull requests made here, so I have a hard time separating the code meant to be reviewed.

Please write issues about future cleaning.

Reply:
Issue written!